### PR TITLE
[4.0] Custom Elements fix - editor none

### PR DESCRIPTION
**Pull request by dgrammatiko**
Pull Request for Issue https://twitter.com/jaffathecake/status/1011214372696395777.

#### TL;DR
The web is a streaming media thus chunks of code served from the server might contain elements partially thus it shouldn't be assumed that the browsers will render the custom elements with all their childrens nodes synchronously. We overcome this by setting an observer to watch the state of the child nodes.  

### Testing Instructions
Apply this patch, change the editor to none and ensure that everything still works as before

Also you can check the lazy loading of the textarea element (what this PR is fixing) here:
https://codepen.io/dgrammatiko/pen/pQKjeR
 
screenshot 2018-11-25 at 14 46 35
 
### Expected result
Custom element work in an async mode

### Actual result
Custom element works only in a synchronous mode

### Documentation Changes Required
Nope

### CODE REVIEW

**Review by wilsonge — APPROVED** [review]

**Comment by wilsonge** [comment]
I have tested this item :white_check_mark: successfully on 0543bd41216aa178e38da17d87a8d4216d78886a<br /><br />Editor none still works after patch in com_content (and tested media, page break and article quick icons successfully)<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23169">issues.joomla.org/tracker/joomla-cms/23169</a>.</sub>

**Comment by wilsonge** [comment]
@dgrammatiko looks like this combined with your babel minifying code has broken drone. Can you please investigate :)

**Comment by dgrammatiko** [comment]
@wilsonge done

**Comment by wilsonge** [comment]
Thanks :)
